README.md

```diff
@@ -50,6 +41,7 @@ import os
 import warnings
 import sys
 
+import boto3
```

Comment:
I feel like this code example could be moved to an `examples/` folder in this repository. Having this in the Readme before the instructions to access the dashboard feels strange.